docs/sources/writing-guide/contribute-release-notes/index.md

```diff
@@ -12,25 +12,130 @@ keywords:
+
+Consider adding an image, though be aware that we will not maintain them, as the document reflect a "point in time." If you need to add an image, refer to [Image, diagram, and screenshot guidelines]({{< relref "../image-guidelines/" >}}).
+
+### When the prior release is complete
+
+Complete the following steps within one week after the prior release goes out.
+
+1. The Docs Squad identifies the technical writer responsible for overseeing the What’s New content development process for the upcoming release.
+1. The identified technical writer cuts a branch and creates a PR with an empty What's New doc to be populated with the What’s New content for the next release. 
+1. The technical writer then pins a link to the [version-number-release-gtm](https://raintank-corp.slack.com/archives/C02L8GQJGBW) Slack channel.
+
+   > **Note:** A Product Manager (PM) is responsible for update the version number of that channel after a release goes out. 
+   
+1. The technical writer posts in the Slack channel that the What's New PR is ready for contributions. 
+1. The PM or PMM on point communicates with eng squads without a PM resource about who’s on point for What’s New PR reviews from within the PM and PMM orgs.
+
+   Refer to [Grafana Releases and Release Dates](https://grafana-intranet--simpplr.visualforce.com/apex/simpplr__app?u=/site/a145f000001dCXBAA2/page/a125f000001AXrwAAG) for a list of release dates and go-to-market coordinators.
+
+### Throughout a release
+
+A Grafana release cycle is typically 6-8 weeks, and during that time, developers are empowered to add to the What's New document incrementally. To see a list of release dates and assigned PMs and PMMs, refer to [Grafana Releases and Release Dates](https://grafana-intranet--simpplr.visualforce.com/apex/simpplr__app?u=/site/a145f000001dCXBAA2/page/a125f000001AXrwAAG).
+
+Complete the following steps throughout a release cycle:
+
+1. Dev teams add commits directly against the branch.
+
+   > **Note:** As an alternative, Dev teams can cut PRs against the branch, add What’s New content, collaborate with Reviewers, and merge the PR into the What's New branch.
+
+1. The technical writer periodically checks the What's New PR to determine how much content has been added.
+
+At this point, the larger Marketing org can review the What’s New content in the branch.
+
+### At the end of a release
+
+Complete the following steps in the days leading up to the release:
+
+1. Five days before the GA release, the PM or PMM who is running the release sends out a **Last Call** to dev teams to get their What’s New content in.
+
+1. Two days before the release, What’s New content is frozen.
+
+1. On release day, the technical writer merges the What’s New branch into main and backports the doc to the feature version branch.
```

Comment:
Unclear on why we're merging at this step and again in the last step. I think this all belongs in the last step.

Reply:
ah, neglected to take out that step. Thanks for the catch.

Reply:
actually, I need to move that step.